CSharpFunctionalExtensions/Result/Dto/ResultDtoJsonConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿#nullable enable
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace CSharpFunctionalExtensions
+{
+    public class ResultDtoJsonConverter : JsonConverter<Result>
```

Comment:
Since `ResultDto` is internal, it makes sence to rename it to just `ResultJsonConverter`

Reply:
Renamed